index.html

```diff
@@ -35,16 +39,20 @@
     </div>
     <div id="navbar" class="menu navbar-collapse collapse navbar-right">
       <ul>
-        <li>Great for <a href="#developers">DEVELOPERS</a>, <a href="#store-managers">MANAGERS</a> &amp; <a href="#customers">CUSTOMERS</a></strong></li>
+        <li class="gh-widget">
+          <a href="http://github.com/mirumee/saleor" target="_blank">
```

Comment:
Any outgoing link to 3th party sites should have the `rel="nofollow"` attribute.

Reply:
:+1: